personality_insights/BlueMix/controllers/application_controller.rb

```diff
@@ -0,0 +1,69 @@
+$:.unshift(File.expand_path('../../lib', __FILE__))
+ 
+require 'sinatra/base'
+require 'json'
+require 'excon'
+require 'twitter'
+
+
+class ApplicationController < Sinatra::Base
+ 
+  helpers ApiHelpers
+
+  set :views, File.expand_path('../../views', __FILE__)
+  set :public_folder, File.expand_path('../../public/', __FILE__)
+
+  
+  set :twitter_api_key, "ENTER API KEY HERE"
+  set :twitter_api_secret, "ENTER API SECRET HERE"  # you shouldn't really do this way since it's not that
```

Comment:
Do you have an alternative approach in mind that you could describe briefly in a comment?